roundabout-core/src/main/java/at/fhv/itm3/s2/roundabout/RunSimulation.java

```diff
@@ -0,0 +1,39 @@
+package at.fhv.itm3.s2.roundabout;
+
+import at.fhv.itm14.trafsim.persistence.model.ScenarioDTO;
+import at.fhv.itm14.trafsim.util.ScenarioReadException;
+import at.fhv.itm14.trafsim.util.XMLParser;
+import at.fhv.itm3.s2.roundabout.util.ConfigParser;
+import at.fhv.itm3.s2.roundabout.util.ConfigParserException;
+import at.fhv.itm3.s2.roundabout.util.ILogger;
+import at.fhv.itm3.s2.roundabout.util.dto.RoundAboutConfig;
+import org.apache.logging.log4j.LogManager;
+
+public class RunSimulation implements ILogger {
+    private static final String DEFAULT_ROUNDABOUT_CONFIG_FILENAME = "roundabout.xml";
+    private static final String DEFAULT_TRAFSIM_CONFIG_FILENAME = "trafsim.xml";
+
+    public static void main(String[] args){
+        String roundaboutConfigFileName = getArgOrDefault(args, 0, DEFAULT_ROUNDABOUT_CONFIG_FILENAME);
+        try {
+            RoundAboutConfig roundAboutConfig = ConfigParser.loadConfig(roundaboutConfigFileName);
```

Comment:
the loaded config isn't used anywhere in this class. Is this just a test class or will it be used in future?

Reply:
WIP = work in progress